content/_data/changelogs/weekly.yml

```diff
@@ -93,6 +93,10 @@
       pull: 2683
     - type: rfe
       message: >
+        Searching in the <code>Build History</code> widget takes into account user preferences (case sensitivity by default).
```

Comment:
`Searching in the <code>Build History</code> widget takes into account user case sensitivity preferences into account`?

Maybe mentioning the default is not required, epecially since you want to change it in https://github.com/jenkinsci/jenkins/pull/2801